core/src/main/java/com/equo/application/client/EquoApp.java

```diff
@@ -93,6 +93,8 @@ public EquoApp setNewTabPageURL(String url) {
    * @param appName Represents the name of the application.
    */
   public static void setAppName(String appName) {
+    CUSTOM_URL = appName;
+    CLASSPATH_URI = String.format("%s://%s/", CLASSPATH_SCHEME, CUSTOM_URL);
```

Comment:
It should come after this line https://github.com/equodev/sdk/pull/45/files#diff-8f329fd1c4fea3dbd2da1d9aff8e3e34298cada502e1dd09ce9dcb781c8761eeR106

Reply:
The appId is sanitized.